website/src/examples/editable-layers/editor.tsx

```diff
@@ -20,7 +19,10 @@ class EditorDemo extends Component {
 
     return (
       <BrowserOnly>
-        {() => <App {...otherProps} />}
+      {()=>{
+        const App = require('../../../../examples/editable-layers/editor/example');
```

Comment:
I guess this works. I'd feel better if we pinned down and removed the dependency, but I guess we have bigger fish to fry. 

A comment explaining why we are doing such an unusual import would have been nice.

Reply:
Resolved by
- #94